elements/pfe-accordion/test/old-test/pfe-accordion_vue_test.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
     <script src="/components/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
```

Comment:
You might get the polyfill migration to CDN by updating with the latest from master.